src/assets/modules/data/README.md

```diff
@@ -0,0 +1,35 @@
+# Data module
+## Quick Summary
+Data is a module that lets you fetch data from CKAN (data.qld.gov.au) through PostgreSQL queries 
+
+## Dependencies
```

Comment:
I believe it is still not a stable product. We have to update loader to use all built modules, create example pages to use modules, test it on existing pages